### DISCUSSION TITLE
libntirpc next does not compile.

### DISCUSSION THREAD

**Discussion by mivry** [nfs-ganesha/nfs-ganesha#1160] [General]
Trying to update nfs_ganesha with latest libntirpc. nfs_ganesha build fails with the following error:

nfs-ganesha/src/libntirpc/src/svc_vc.c:547:17: error: implicit declaration of function ‘svc_get_port’ [-Werror=implicit-function-declaration]
  547 |                 svc_get_port(newxprt->xp_local.nb.buf));
  
libntirpc commit includes a log message that calls to undefined function svc_get_port.

**Comment by rojingeorge** [comment]
@mivry - Apologies :-(
Have fixed and submitted the patch...
https://github.com/nfs-ganesha/ntirpc/pull/309

Please cherry pick the same, until merged...

@dang - Please help to merge in and Sorry again...

**Comment by dang** [comment]
Merged.
